db/seeds.rb

```diff
@@ -5,3 +5,27 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+require_relative '../spec/support/marc_fixture_seed_fetcher'
+
+Upload.skip_callback(:commit, :after, :perform_extract_marc_record_metadata_job)
+
+Settings.marc_fixture_seeds.organizations.each do |org|
+  organization = Organization.create(name: org, slug: org.downcase)
+
+  MarcFixtureSeedFetcher.fetch_uploads(org.downcase) do |upload, files|
+    upload = organization.default_stream.uploads.build(name: upload['name'])
+    files.map do |file|
+      io = URI.open(file['download_url'], 'Authorization' => "Bearer #{Settings.marc_fixture_seeds.token}")
```

Comment:
I could theoretically merge this into the file data in `MarcFixtureSeedFetcher` so we don't have to have knowledge of how to do authorization here, but 🤷